src/main/kotlin/coffeebot/commands/Milton.kt

```diff
@@ -0,0 +1,58 @@
+package coffeebot.commands
+
+import coffeebot.message.Valid
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.okhttp.OkHttp
+import io.ktor.client.features.HttpTimeout
+import io.ktor.client.features.json.GsonSerializer
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.client.request.post
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.HttpStatusCode
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+import java.lang.IllegalStateException
+import java.net.MalformedURLException
+import java.net.URL
+
+class MiltonClient {
+    private val httpClient = HttpClient(OkHttp) {
+        install(JsonFeature) {
+            serializer = GsonSerializer()
+        }
+        install(HttpTimeout) {
+            requestTimeoutMillis = 45_000
+            socketTimeoutMillis = 45_000
+        }
+    }
+    private val miltonHost = "https://milton.terbium.io"
+
+    fun index(url: URL): Boolean {
+        val result = runBlocking(Dispatchers.IO) {
+            httpClient.post<HttpResponse>("$miltonHost/save?url=${url}")
+        }
+        println(result)
+        return result.status == HttpStatusCode.OK
+    }
+}
+
+private val miltonClient = MiltonClient()
+private val urlRegex = "https?://[\\w\\d:#@%/;$()~_?+-=\\\\.&]*".toRegex(RegexOption.IGNORE_CASE)
+
+val milton = PassiveCommand { message ->
+    val matches = urlRegex.findAll(message.contents).toList()
+    val results = matches.map { match ->
+        val urlValue = match.value
+        println(urlValue)
```

Comment:
I'm fine with leaving print statements as log statements. This is intent, right? (not relic of debugging)

Reply:
No it's clearly the rusty remains of printf-based debugging :p